### ISSUE TITLE
Sending emails from a dockerized, self-hosted instance fails when using gmail smtp 

### ISSUE THREAD

**Issue by teodoryantcheff** [outline/outline#4698]
**To Reproduce**

1. Invite a new user to force sending an email
2. Look at `docker logs -f <container_name>`
3. See error

From the outline log : 
```
outline_1   | {"error":"Server terminates connection. response=421 4.7.0 Try again later, closing connection. (EHLO) s12-20020a170902ea0c00b00192dbd5b753sm1113729plg.113 - gsmtp: 421 4.7.0 Try again later, closing connection. (EHLO) s12-20020a170902ea0c00b00192dbd5b753sm1113729plg.113 - gsmtp","stack":"Error: Server terminates connection. response=421 4.7.0 Try again later, closing connection. (EHLO) s12-20020a170902ea0c00b00192dbd5b753sm1113729plg.113 - gsmtp: 421 4.7.0 Try again later, closing connection. (EHLO) s12-20020a170902ea0c00b00192dbd5b753sm1113729plg.113 - gsmtp\n    at SMTPConnection._actionEHLO (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:1229:27)\n    at SMTPConnection._processResponse (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:932:20)\n    at SMTPConnection._onData (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:739:14)\n    at TLSSocket.SMTPConnection._onSocketData (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:189:44)\n    at TLSSocket.emit (node:events:526:28)\n    at TLSSocket.emit (node:domain:475:12)\n    at addChunk (node:internal/streams/readable:315:12)\n    at readableAddChunk (node:internal/streams/readable:289:9)\n    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)\n    at TLSWrap.onStreamRead (node:internal/stream_base_commons:190:23)","level":"error","message":"Error processing task in EmailTask"}
```

I.e - `421 4.7.0 Try again later, closing connection. (EHLO)`

Environment:
```
SMTP_HOST=smtp-relay.gmail.com | smtp.gmail.com
SMTP_PORT=465 | 587
SMTP_SECURE=true | false
.
SMTP_USERNAME=<valid username>
SMTP_PASSWORD=<valid password>
.
DEBUG=email | *
```

'|' separated are all the options I have tried.

`smtp-relay.gmail.com` is the relay we also use to send other email with the credentials from above, and obviously we do successfully send those emails.
We have enabled 'allow less secure...` on our google workspace.

outline version `0.67.1`.

I suspect that this ticket is superfluous, as likely this is a nodemailer problem. As suggested [here](https://stackoverflow.com/questions/70430898/how-to-fix-nodemailer-problem-error-server-terminates-connection-response-421).

**Comment by tommoor** [comment]
According to this answer we'd need to add another config property so that `name` can be passed through to gmail, you're using Google Workspace yea?

https://stackoverflow.com/a/74820750

**Comment by tommoor** [comment]
Should be resolved by e3db7455b33252e35b17a5c3520b21194b2b968f

**Comment by buuhvprojects** [comment]
> According to this answer we'd need to add another config property so that `name` can be passed through to gmail upon SMTP connection, you're using Google Workspace yea? In which case name would be set to your hosted domain.
> 
> https://stackoverflow.com/a/74820750

this worked for me, thanks. Now i can run nodemailer inside docker container
